crates/shared/src/sources/balancer_v2/swap/error.rs

```diff
@@ -65,6 +65,6 @@ mod tests {
     #[test]
     #[should_panic]
     fn error_parse_fail() {
-        Error::from("Poop");
+        let _ = Error::from("Poop");
```

Comment:
Another lint error that was already fixed on `main` and will go away once rebased.

Reply:
Ha ha. Poop.